src/SDKs/DataFactoryV1Extended/DataFactory.Tests/DataFactory.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>DataFactory.Tests</PackageId>
+    <Description>DataFactory.Tests Class Library</Description>
+    <AssemblyName>DataFactory.Tests</AssemblyName>
+    <VersionPrefix>4.13.0</VersionPrefix>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0-preview-20170425-07" />
```

Comment:
@nicolaszhang these references will be redundant. They are already been included in all test projects.
We do not want common dependencies to be provided twice, because it is then easy to upgrade these references from a common place.